config/rbac/metrics_auth_role_binding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: proxy-rolebinding
+  name: metrics-auth-rolebinding
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: proxy-role
+  name: metrics-auth-role
 subjects:
-- kind: ServiceAccount
-  name: default
-  namespace: system
+  - kind: ServiceAccount
+    name: controller-manager
```

Comment:
Who creates the `controller-manager` service account?
The manager.yaml doesn't specify `ServiceAccount` specifically.

Reply:
Hi @bpradipt! Thanks for the comment!
If I understand correctly this service account was introduced in v1.5.0 sdk version (https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.5.0/#gov3-add-a-systemcontroller-manager-serviceaccount-to-your-project)
Do you know the "default" was kept by purpose? Or I can change it to new "controller-manager" SA in other rolebinding as well?

Reply:
@balintTobik I think we just didn't remove it since OSC operator was always installed in a specific namespace.